dockerfiles/tasks.py

```diff
@@ -157,3 +160,39 @@ def buildassets(c):
     """Build all assets for the application and push them to backend storage"""
     c.run(f'docker-compose -f {DOCKER_COMPOSE_ASSETS} run --rm assets bash -c "npm ci && node_modules/bower/bin/bower --allow-root update && npm run build"', pty=True)
     c.run(f'{DOCKER_COMPOSE_COMMAND} run --rm web python3 manage.py collectstatic --noinput', pty=True)
+
+
+@task(help={
+    'action': 'Action to realize on Transifex ("pull" or "push")',
+})
+def translations(c, action):
+
+    if action not in ('pull', 'push'):
+        print(f'Action passed ("{action}") not supported. Use "pull" or "push".')
+        sys.exit(1)
+
+    transifex_token = os.environ.get('TRANSIFEX_TOKEN', None)
+    if not transifex_token:
+        print('You need to export TRANSIFEX_TOKEN environment variable.')
+        sys.exit(1)
+
+    if sys.platform in ('linux', 'linux2'):
+        download_file = 'https://github.com/transifex/cli/releases/download/v1.1.0/tx-linux-amd64.tar.gz'
+    elif sys.platform == 'darwin':
+        download_file = 'https://github.com/transifex/cli/releases/download/v1.1.0/tx-darwin-amd64.tar.gz'
```

Comment:
Isn't this file just used from inside the Docker container? It seems to me that we don't need to support Mac OS here if so, the platform should always be Linux

Reply:
Oh, yeah. You are right. I was trying a different approach first and then moved the code to here. I will remove this now.